### PR TITLE
Criu container fat

### PR DESCRIPTION
**Pull request by sebratton**

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

File: dev/io.openliberty.checkpoint_fat_container/publish/servers/checkpoint.fat.container/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info:criu=all
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_container/publish/servers/checkpoint.fat.container/bootstrap.properties]: In my latest PR #20399 I was going to change the trace group to `checkpoint`

File: dev/io.openliberty.checkpoint_fat_container/publish/servers/checkpoint.fat.container/server.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    <featureManager>
+        <feature>servlet-4.0</feature>
+        <feature>checkpoint-1.0</feature>
+    </featureManager>
+    
+
+    <httpEndpoint id="defaultHttpEndpoint"
+                  host="*"
+                  httpPort="9080"
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_container/publish/servers/checkpoint.fat.container/server.xml:21]: Do the ports get configured with the test ports?  Or do we always have the container use 9080?

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Review by tjwatson — APPROVED** [review]
I approve.  Note that the trace group has changed to `checkpoint`.

**Comment by sebratton** [comment]
#libbybot

**Pull request merged**